src/complex/Filter/Output.hpp

```diff
@@ -107,13 +107,30 @@ class COMPLEX_EXPORT IDataCreationAction : public IDataAction
   DataPath m_CreatedPath;
 };
 
+/**
+ * @brief
+ */
+struct COMPLEX_EXPORT DataModifiedAction
```

Comment:
We might want to call this change "Action" in the name to avoid confusion with actual action classes.

Reply:
? We _don't_ want to call it "Action" ?

Reply:
We use it similarly to actions, but it doesn't derive from `IDataAction`. So I'm not sure of the best thing to call it.